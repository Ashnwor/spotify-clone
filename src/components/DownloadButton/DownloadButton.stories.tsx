import React from 'react';
import DownloadButton from './DownloadButton';

export default {
  title: 'DownloadButton',
};

export const downloadButton = () => <DownloadButton />;
