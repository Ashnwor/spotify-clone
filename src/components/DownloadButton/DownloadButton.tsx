import React from 'react';
import { DownloadIcon } from '../icons';
import './DownloadButton.scss';

const DownloadButton: React.FC = () => {
  return (
    <div className="download-button-container">
      <div className="download-button-inner-container">
        <div className="download-button-icon-container">
          <DownloadIcon className="download-button-icon" />
        </div>
        <span className="download-button-text">Download App</span>
      </div>
    </div>
  );
};

export default DownloadButton;
