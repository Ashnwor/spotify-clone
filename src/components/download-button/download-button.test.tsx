import React from 'react';
import { shallow } from 'enzyme';
import DownloadButton from './download-button';

it('runs correctly', () => {
  const wrapper = shallow(<DownloadButton />);
  expect(wrapper).toMatchSnapshot();
});
