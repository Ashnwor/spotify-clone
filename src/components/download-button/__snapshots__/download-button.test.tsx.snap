// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`runs correctly 1`] = `
<div
  className="download-button-container"
>
  <div
    className="download-button-inner-container"
  >
    <div
      className="download-button-icon-container"
    >
      <SvgDownloadIcon
        className="download-button-icon"
      />
    </div>
    <span
      className="download-button-text"
    >
      Download App
    </span>
  </div>
</div>
`;
